### DISCUSSION TITLE
Mohist 1.12.2 | 1.16.5 Support

### DISCUSSION THREAD

**Discussion by MrDefined** [Zrips/CMI#6806] [Feature Requests]
I run a Pixelmon Reforged server and recently switched from spongeforge to mohist. I absolutely love CMI and use it in every single one of my servers ranging from Survival, OneBlock, and more.

I also have implemented CMI on my Pixelmon server, which runs 1.12.2 Mohist. The only issue is this plugin doesn't really support mohist very well. Or honestly the other way around lol..

I would, as well as others in the pixelmon community would LOVE to have mohist support added to this amazing plugin, right now there are a bunch of smaller issues / bugs like our holograms randomly disappearing or switching places when loading in for a player. Sometimes they are there and sometimes not.

PLEASE add mohist support lol :D

**Comment by MrCrazys** [comment]
CMI does not officially support forks of Spigot/Paper or modded/hybrid server softwares mainly due to potential differences in the base code. If it works, it works. If it doesn't, unfortunately it isn't and probably will never be supported. It is clearly outlined on [https://www.zrips.net/faq](https://www.zrips.net/faq).

**Comment by mrfloris** [comment]
It's fine to request though. 

But I agree, the plugin is rather big and the modded servers like Mohist, Magma, and whatever are just too different. I can't imagine having to put all that time into supporting modded when their % of usage is (so) low.

I have no say in yes/no on this topic, but in my opinion, I suspect we shouldn't expect support for it any time soon.
